crates/wassette/src/lib.rs

```diff
@@ -1233,6 +1233,12 @@ impl LifecycleManager {
         component_id: &str,
         secrets: &[(String, String)],
     ) -> Result<()> {
+        // Check if component exists in the plugin directory
+        let component_path = self.component_path(component_id);
+        if !component_path.exists() {
+            bail!("Component not found: {}", component_id);
+        }
```

Comment:
Question: does this work with if wassette CLI is suing `plugin_dir` to change the path from a default one to a user provided one?